client/cmdhf14a.c

```diff
@@ -35,8 +35,13 @@
 static int CmdHelp(const char *Cmd);
 static int waitCmd(uint8_t iLen);
 
+// structure and database for uid -> tagtype lookups 
+typedef struct { 
+	uint8_t uid;
+	char* desc;
+} manufactureName; 
```

Comment:
What is the reason for moving typedef's into the .c file from the header file?

Reply:
Just because it is only needed in cmdhf14a.c and nowhere else.

Reply:
we have a manufacture list in 15 aswell,  one of these days it would be nice to have only one reference. I belive it could be used in more places aswell.  It should be in its own file. 
or what do you think?